DataFrames/Plots/Histogram.js

```diff
@@ -137,6 +139,54 @@ define([
                 win.render();
             };
 
+            /**
+             * Create text for display in info section of the plot
+             * @param {[]} values: list of data values
+             * @returns {String}: info text to display
+             * @private
+             */
+            win._infoText = function(values) {
+
+                var minVal = +1.0e99;
+                var maxVal = -1.0e99;
+                var sum = 0;
+                var count = values.length;
+
+                for (var rowNr = 0; rowNr < count; rowNr++) {
+                    var val = values[rowNr];
+                    if (val<minVal) minVal = val;
+                    if (val>maxVal) maxVal = val;
+                    sum += val;
+                }
+
+                var str = '';
+                str += 'Min: ' + minVal + '<br>';
+                str += 'Max: ' + maxVal + '<br>';
+                str += 'Count: ' + count + '<br>';
+                str += 'Sum: ' + sum + '<br>';
+                var average = sum/count;
+                str += 'Average: ' + average + '<br>';
+
+                var stdev = 0;
+                for (var i=0; i<values.length; i+= 1)
+                    stdev += Math.pow(values[i]-average, 2.0);
+                stdev = Math.sqrt(stdev/values.length);
+                str += 'Stdev: ' + stdev + '<br>';
+
+                function median(values) {
+                    values.sort( function(a,b) {return a - b;} );
```

Comment:
`Array.prototype.sort()` sorts the array in place, mutating the input parameter of the method without the caller knowing it. Probably isn't too much of an issue here, I don't think the caller relies on the order not changing. Nevertheless a good idea to make a shallow copy first before sorting by slicing it (i.e. `array.slice().sort(...)`)